### PR TITLE
📝 Add metadata to workflow executions feature

### PR DESCRIPTION
**Pull request by samyok**

### CODE REVIEW

**Comment by dvargas92495** [comment]
I would follow the steps listed [here](https://github.com/vellum-ai/vellum-client-generator/blob/main/README.md?plain=1#L43-L54) in order to include the schema changes here as well!

File: fern/docs/content/change-log/2024-08.mdx [hunk 1]
```diff
@@ -2,6 +2,19 @@
...
 
+## Add Metadata to Workflow Executions
+
+_August 27th, 2024_
+
+You can now add metadata to your Workflow Executions through the API. This is useful for tracking additional information
+about your executions, such as the source of the request or any other custom data you want to associate with the
+execution.
+
+This metadata is visible in the Workflow Execution Details page in the Vellum UI.
```
Comment (dvargas92495) [inline fern/docs/content/change-log/2024-08.mdx:13]: Agh, I should've clarified, I was hoping to keep the column but just remove the search operators with my comment from here: https://github.com/vellum-ai/vellum/pull/5200/commits/b8932b18b424ef3adb12aadbbe041d3d906686f5#diff-788279b66dffcd471fcabdab1645d5ff78e9c34b9262e041358fcee47edfc82bL540-L552

**Review by dvargas92495 — APPROVED** [review]
We can clean up the column separately - we could wrap this one up for now and 🚢 !

File: fern/docs/content/change-log/2024-08.mdx (hunk 1 above)
Comment (samyok) [reply fern/docs/content/change-log/2024-08.mdx:13]: aw man, I'll add that back

**Pull request merged**